features/product/viewing_product_in_admin_panel/viewing_product_catalog_price_history_within_product_variant.feature

```diff
@@ -0,0 +1,38 @@
+@viewing_products
+Feature: Seeing catalog price history within a variant
+    In order to be aware of historical variant prices
+    As an Administrator
+    I want to browse the catalog price history of a specific variant
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a "Wyborowa Vodka" configurable product
+        And the product "Wyborowa Vodka" has a "Wyborowa Vodka Exquisite" variant priced at "$40.00"
+        And the product "Wyborowa Vodka" has a "Wyborowa Vodka Lemon" variant priced at "$10.00"
+        And there is a catalog promotion with "winter_sale" code and "Winter sale" name
+        And the catalog promotion "Winter sale" is available in "United States"
+        And it applies on "Wyborowa Vodka Exquisite" variant
+        And it reduces price by "13%"
+        And it is enabled
+        And there is also a catalog promotion with "christmas_sale" code and "Christmas sale" name
+        And the catalog promotion "Christmas sale" is available in "United States"
+        And it applies on "Wyborowa Vodka Lemon" variant
+        And it reduces price by "37%"
+        And it is enabled
+        And the "Wyborowa Vodka Exquisite" variant is now priced at "$45.00" and originally priced at "$15.00"
+        And I am logged in as an administrator
+        And I disable "Winter sale" catalog promotion
+
+    @todo
+    Scenario: Seeing the catalog price history of all variants of a product
+        When I access "Wyborowa Vodka" product
+        And I go to the "Wyborowa Vodka Exquisite" variant price history
+        Then I should see 4 logs entries in the catalog price history for the "Wyborowa Vodka" variant
+        And there should be a log entry on the 1st position with the "$45.00" selling price, "$15.00" original price and datetime of the price change
+        And there should be a log entry on the 2st position with the "$39.15" selling price, "$15.00" original price and datetime of the price change
```

Comment:
If I'm not mistaken of course, catalog promotions don't work that way, if the original price is set then only that will be taken into account when calculating the new price, so here it should be: 
```suggestion
        And there should be a log entry on the 1st position with the "$15.00" selling price, "$15.00" original price and datetime of the price change
        And there should be a log entry on the 2st position with the "$13.05" selling price, "$15.00" original price and datetime of the price change
```

Reply:
As discussed I added a few more scenarios including scenarios with and without an original price and with more than one catalog promotion applied on a variant.